chapters/samplers.txt

```diff
@@ -435,6 +435,13 @@ include::{generated}/api/enums/VkBorderColor.txt[]
 These colors are described in detail in <<textures-texel-replacement, Texel
 Replacement>>.
 
+ifdef::VK_EXT_custom_border_color[]
```

Comment:
This should move up before the "These colors..." paragraph, so it's part of the list.